feature-libs/organization/administration/core/services/b2b-user.service.ts

```diff
@@ -63,7 +62,7 @@ export class B2BUserService {
   private getB2BUserValue(orgCustomerId: string): Observable<B2BUser> {
     return this.store
       .select(getB2BUserValue(orgCustomerId))
-      .pipe(filter(Boolean));
+      .pipe(filter((value) => Boolean(value)));
```

Comment:
For readability can we call the var `b2bUser` (I think that's what it is) instead of `value`?